employee.rb

```diff
@@ -0,0 +1,20 @@
+require("./assignment3.rb")
```

Comment:
this line is not needed I guess.

Reply:
We require this as it will not show any output

Reply:
removed